vipps-login-api-faq.md

```diff
@@ -156,6 +157,9 @@ electronic ID, e.g  BankID.
 
 Merchants need to apply for access to NNIN separately by sending an email to AccessUserInfo@vipps.no. In the email you should specify merchant name and organization number and name of sales unit from [VippsPortalen](https://portal.vipps.no). You also need to provide information on how you plan to use the NNIN, the legal requirement and/or the reason why you need to use NNIN to achieve required user identification.
 
+## Who can get access to account numbers and how?
```

Comment:
We can add the text: 
Access to account numbers is a payed service. Merchants need to order access to account numbers separately by sending an email to AccessUserInfo@vipps.no. In the email you should specify merchant name and organization number and name of sales unit from [VippsPortalen](https://portal.vipps.no). You also need to provide information on how you plan to use the account number.

Reply:
Added this text.